processing/src/main/java/io/druid/segment/incremental/OffheapIncrementalIndex.java

```diff
@@ -92,6 +93,26 @@ public OffheapIncrementalIndex(
   }
 
   public OffheapIncrementalIndex(
```

Comment:
Yes. Also having 4 boolean parameters in a row on the call sites is dangerous (to accidentally switch them) and not readable, consider replacing one or two of them with  enums of two options.